include/private/hashmap.h

```diff
@@ -292,4 +292,15 @@ static inline uint64_t hash_fnv1_iterate_string(uint64_t h, const char *str)
     return h;
 }
 
+static inline uint64_t hash_fnv1_iterate_data(uint64_t h, const void *ptr_, size_t size)
+{
+    const uint8_t *ptr = ptr_;
+    size_t i;
+
+    for (i = 0; i < size; i++)
+        h = hash_fnv1_iterate_u8(h, ptr[i]);
```

Comment:
Given that we use this to hash blobs, might be worth doing this per-dword rather than per-byte and then just gather the last couple of bytes by hand if the size isn't aligned to 4?

Reply:
There's also alignment of the pointer to consider.

Reply:
Removed function.